app/assets/javascripts/ang/controllers/observation_search.js

```diff
@@ -90,6 +90,16 @@ function( ObservationsFactory, PlacesFactory, TaxaFactory, shared, $scope, $root
   $scope.dragRectDeltaXInPx = null;
   $scope.dragRectDeltaYInPx = null;
   $scope.dragCircleDeltaInKm = null;
+  $scope.dragIconMarker = new google.maps.Marker({
+    position: { lat: 0, lng: 0 },
+    icon: {
+      anchor: { x: 12, y: 12 },
+      url: "assets/drag-icon.png",
```

Comment:
Hard-coding the path to a file in the asset pipeline won't work in production. We usually handle this by

1. Renaming the file `observation_search.js.erb`. This will cause the file to get run through the ERB interpreter when the assets get compiled.
2. Using the asset helper to insert the correct URL
    ```javascript
    url: "<%= asset_path( "drag-icon.png" ) %>",
    ```

Also, can we use an SVG here? I think Abhas sent you one. Not sure if it works with Google, but we can definitely serve an SVG assets just like we can a PNG. We use SVGs for a lot of our icons and illustrations.

Reply:
Thanks @kueda, updated.